src/Annotations/AppKernel.php

```diff
@@ -0,0 +1,292 @@
+use Psr\Container\ContainerInterface;
+use Silex\Application;
+use Silex\Provider\HttpFragmentServiceProvider;
+use Silex\Provider\ServiceControllerServiceProvider;
+use Silex\Provider\TwigServiceProvider;
+use Silex\Provider\VarDumperServiceProvider;
+use Silex\Provider\WebProfilerServiceProvider;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+use Symfony\Component\HttpKernel\TerminableInterface;
+
+final class AppKernel implements ContainerInterface, HttpKernelInterface, TerminableInterface
+{
+    private $app;
+
+    public function __construct(string $environment = 'dev')
+    {
+        $configFile = __DIR__.'/../../config.php';
+        $config = array_merge(file_exists($configFile) ? require $configFile : [], require __DIR__."/../../config/{$environment}.php");
+
+        $this->app = new Application([
+            'debug' => $config['debug'] ?? false,
+            'ttl' => $config['ttl'] ?? 300,
+            'logging.path' => $config['logging.path'] ?? __DIR__.'/../../var/logs',
+            'logging.level' => $config['logging.level'] ?? Logger::INFO,
+            'api.url' => $config['api_url'] ?? 'https://api.elifesciences.org/',
+            'api.requests_batch' => $config['api_requests_batch'] ?? 10,
+            'process_memory_limit' => $config['process_memory_limit'] ?? 256,
+            'aws' => $config['aws'] + [
+                'queue_name' => 'annotations--queue',
+                'queue_message_default_type' => 'profile',
+                'credential_file' => true,
+                'region' => 'us-east-1',
+                'endpoint' => 'http://localhost:4100',
+             ],
+            'hypothesis' => $config['hypothesis'] + [
+                'api_url' => 'https://hypothes.is/api/',
+                'client_id' => '',
+                'secret_key' => '',
+                'authority' => '',
+            ],
+        ]);
+
+        $this->app->register(new ApiProblemProvider());
+        $this->app->register(new PingControllerProvider());
+
+        if ($this->app['debug']) {
+            $this->app->register(new VarDumperServiceProvider());
+            $this->app->register(new HttpFragmentServiceProvider());
+            $this->app->register(new ServiceControllerServiceProvider());
+            $this->app->register(new TwigServiceProvider());
+            $this->app->register(new WebProfilerServiceProvider(), [
+                'profiler.cache_dir' => __DIR__.'/../../var/cache/profiler',
+                'profiler.mount_prefix' => '/_profiler',
+            ]);
+        }
+
+        // General cache.
+        $this->app['cache'] = function () {
+            return new FilesystemCache(__DIR__.'/../../var/cache');
+        };
+
+        $this->app['logger'] = function (Application $app) {
+            $factory = new LoggingFactory($app['logging.path'], 'annotations-api', $app['logging.level']);
```

Comment:
would just say `annotations` as 2nd argument, since this logger also covers the cli processes